src/ui/components/LoadingText/styles.scss

```diff
@@ -28,8 +28,9 @@
   display: inline-block;
   height: 1rem;
   line-height: 1;
-  margin: 0;
+  margin-left: 300px; // added new nisamova
```

Comment:
I don't think we need a comment here. In addition to that, we should likely fix the issue by changing the structure of the HTML, not just the style.